src/client/localplayer.h

```diff
@@ -45,7 +46,7 @@ enum class LocalPlayerAnimation
 class LocalPlayer : public Player
 {
 public:
-	LocalPlayer(Client *client, const char *name);
+	LocalPlayer(Client *client, const std::string name);
```

Comment:
this change should at least be a separate commit

Reply:
Aww, thought I could get away with taking the lazy way out. Done.